server/db/models/review.js

```diff
@@ -1,38 +1,45 @@
 const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
 
-const reviewSchema = new Schema({
-  course: {
-    _id: false,
-    type: Schema.Types.ObjectId,
-    ref: "Course",
-    index: true,
-    required: true
+const reviewSchema = new Schema(
```

Comment:
`reviewSchema` will be changed to `recordsSchema` to add the support for Questions. Refer #10

Reply:
Lets handle it after #10 is merged?